### ISSUE TITLE
Delivery Address

### ISSUE THREAD

**Issue by rashadisrazzi** [datakund/amazon-api-datakund#1]
Currently there's a delivery address selection page between the buy and payment. Any way to select default address or create a new one automatically? Selecting the default should be easy enough but I don't see any methods already or the source code to try and code it myself

**Comment by vatsalshah1990** [comment]
Yeah I'm stuck with the same issue. One click buy would've worked as well but its not available as an option for Amazon India. Need access to the code or a function to select the default address.
